### PR TITLE
chore: add IEC units for data rate

### PR DESCRIPTION
**Pull request by YiniXu9506**
- Add IEC units for Data rate
- Update id of Data rate in SI units

### CODE REVIEW

**Comment by YiniXu9506** [comment]
@baurine PTAL~

File: src/valueFormats/valueFormats.test.ts [hunk 1]
```diff
@@ -105,4 +105,48 @@ describe('valueFormats', () => {
...
+    it('Bps should handle number correctly if scaledDecimals is null', () => {
+      expect(getValueFormat('Bps')(1024.33, 2, null)).toBe('1.00 KiBs');
+    });
+  });
+
+  describe('Data Rate (Metric) format ', () => {
+    it('decBps should format number correctly if decimal is null', () => {
+      expect(getValueFormat('decBps')(1000)).toBe('1 kBs');
+      expect(getValueFormat('decBps')(1050)).toBe('1 kBs');
+      expect(getValueFormat('decBps')(100000)).toBe('100 kBs');
```
Comment (baurine) [inline src/valueFormats/valueFormats.test.ts]: Should be `kBs` or `KBs`?

**Review by baurine — APPROVED** [review]
LGTM

File: src/valueFormats/valueFormats.test.ts (hunk 1 above)
Comment (YiniXu9506) [reply src/valueFormats/valueFormats.test.ts]: The metric standard is kBs, and IEC standard is KiBs.

**Pull request merged**